ansible/vars/topo_mgmttor.yml

```diff
@@ -0,0 +1,230 @@
+topology:
+  host_interfaces:
```

Comment:
We might not need host interfaces for mgmt network.
The mgmt network is more like T1 topo, the upstream is mgmt leaf router and the downstream is managed router (T0/T1/...)